plover/assets/dict_unsorted.json

```diff
@@ -121866,7 +121866,6 @@
 "KWRA/AUL": "y'all",
 "KWRAOU/AUL": "y'all",
 "KWRAB/PWA/TKAB/PWA": "yabba dabba",
-"KWRAT": "yacht",
```

Comment:
The reason I've removed KWRAT = yacht is because it conflicts with an older entry for "{^@^}". Since the latter entry was in the dictionary long before the "yacht" entry I decided to remove the yacht one.